src/app/joinus/page.tsx

```diff
@@ -0,0 +1,31 @@
+import JoinUsCard from "@/components/joinus/joincards";
+import { HiOutlineUserGroup } from "react-icons/hi2";
+import { Stethoscope } from "lucide-react";
+import { HandHeart } from "lucide-react";
+
+const JoinUs = () => {
+  return (
+    <div className="bg-naama-blue-300 flex justify-center p-10">
+      <JoinUsCard
+        title="Mentorship and Networking"
+        description="Gain access to a strong network of healthcare professionals and
+        mentors who can guide you on your academic and career path."
+        icon={<HiOutlineUserGroup className="text-naama-blue-300 h-10 w-10" />}
```

Comment:
```suggestion
        icon={<HiOutlineUserGroup" />}
```

move these to inside the component as all of them are the same

Reply:
same for other JoinUsCards